phpseclib/Net/SFTP.php

```diff
@@ -1337,13 +1290,9 @@ public function truncate($filename, $new_size)
      *
      * If the file does not exist, it will be created.
      *
-     * @param string $filename
-     * @param int $time
-     * @param int $atime
-     * @throws \UnexpectedValueException on receipt of unexpected packets
-     * @return bool
+     *@throws \UnexpectedValueException on receipt of unexpected packets
```

Comment:
Seems like a space should be added here as well?

Reply:
Fixed all